config.def.h

```diff
@@ -1,6 +1,13 @@
 /* See LICENSE file for copyright and license details. */
 
 /* appearance options */
+
+/* wallpaper */
+static const bool restorewallpaper 	= false; 	/* when enabled, the wallpapercmd command is run */
```

Comment:
This seems like a hack and doesn't feel like it belongs here. Wouldn't be be better to provide a sample `.xinitrc` (or some other scripts) to achieve this same thing?